torch/_dynamo/symbolic_convert.py

```diff
@@ -871,6 +871,98 @@ def _missing(opname, *args):
         cls.dispatch_table = [dispatch_table.get(i) for i in range(2**8)]
 
 
+@dataclasses.dataclass
+class ExceptionStack:
```

Comment:
The ExceptionStack (accepting suggestions for other name) is shared between all InstructionTranslator instances. An exception raised in a generator is reflected in the parent function `exn_vt_stack`.